Assets/Scripts/UI/StatBar.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class StatBar : MonoBehaviour
+{
+    public string SpriteLocation = "Sprites/PlayerHungerStates_Sprite";
+    
+    private int _statusValue = 100;
+
+    private Sprite[] _barStates;
+
+    private Image[] _images;
+
+    private Animator _animator;
+    
+    
+    // Start is called before the first frame update
+    void Start()
+    {
+        _barStates = Resources.LoadAll<Sprite>(SpriteLocation);
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+         
+    }
+
+    public void UpdateBar(int value)
+    {
+        var newValue = value;
```

Comment:
Don't hate "var" anymore but again, it should be consistent. Please use specific types

Reply:
Also why not use value for everything? And _statusValue